azure-client-runtime/src/main/java/com/microsoft/azure/v2/annotations/AzureHost.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.v2.annotations;
```

Comment:
The convention in Java world is to add version numbers to package names so that multiple versions can be used together